lib/ex_twilio/resources/lookup.ex

```diff
@@ -0,0 +1,33 @@
+defmodule ExTwilio.Lookup do
+  @moduledoc """
+    Represents the Lookup Api provided by Twilio
+
+    - [Twilio docs](https://www.twilio.com/docs/api/lookups)
+  """
+  alias ExTwilio.{Parser, Api, Config}
+  alias ExTwilio.UrlGenerator, as: Url
+
+  @base_url "https://lookups.twilio.com/v1/PhoneNumbers/"
+
+  defstruct url: nil,
+            carrier: nil,
+            national_format: nil,
+            phone_number: nil,
+            country_code: nil,
+            add_ons: nil
```

Comment:
I think I would rather see this struct nested inside a submodule, like this:

```elixir
defmodule PhoneNumber do
  defstruct url: nil,
                  carrier: nil,
                  national_format: nil,
                  phone_number: nil,
                  country_code: nil,
                  add_ons: nil
end
```

This way, a list of `ExTwilio.Lookup.PhoneNumber` structs could be returned rather than a list of `ExTwilio.Lookup` structs.